src/gf-platformid

```diff
@@ -41,19 +41,22 @@ coreos_gf_run_mount "${tmp_dest}"
 # * grub config
 # * BLS config (for subsequent config regeneration)
 # First, the grub config.
-if [ "$(coreos_gf exists '/boot/efi')" == 'true' ]; then
-    grubcfg_path=$(coreos_gf glob-expand /boot/efi/EFI/*/grub.cfg)
-else
-    grubcfg_path=/boot/loader/grub.cfg
+# Set grub.cfg in case the arch uses anaconda
+if [ "$arch" == "ppc64le" ]; then
```

Comment:
I need to change this to only exclude s390x, since this is 4.2 and uses anaconda